pkg/minikube/command/exec_runner.go

```diff
@@ -62,19 +63,19 @@ func (*ExecRunner) RunCmd(cmd *exec.Cmd) (*RunResult, error) {
 	start := time.Now()
 	err := cmd.Run()
 	elapsed := time.Since(start)
+
+	if exitError, ok := err.(*exec.ExitError); ok {
+		rr.ExitCode = exitError.ExitCode()
+	}
+	// Decrease log spam
+	if elapsed > (1 * time.Second) {
+		glog.Infof("Completed: %s: (%s)", rr.Command(), elapsed)
+	}
 	if err == nil {
-		// Reduce log spam
-		if elapsed > (1 * time.Second) {
-			glog.Infof("(ExecRunner) Done: %v: (%s)", rr.Command(), elapsed)
-		}
-	} else {
-		if exitError, ok := err.(*exec.ExitError); ok {
-			rr.ExitCode = exitError.ExitCode()
-		}
-		glog.Infof("(ExecRunner) Non-zero exit: %v: %v (%s)\n%s", rr.Command(), err, elapsed, rr.Output())
-		err = errors.Wrapf(err, "command failed: %s\nstdout: %s\nstderr: %s", rr.Command(), rr.Stdout.String(), rr.Stderr.String())
+		return rr, nil
 	}
-	return rr, err
+
```

Comment:
Consider using a way to indent each result.
Not simply \n
Mind that \t would only indent first line.

We need a helper function that scans a string line by line and idents them and returns them back.

Reply:
Clearifying.
For example

Command .....:
Error:
            ----------
Stdout:
             -------
             -------
Stderr:
              ------
              ------

Reply:
And since most of time stderr is more important. Maybe it can even be upper than stdout.

Usually stdout is very long....
If stdout is very long. Maybe the CMD should be repeated front of stderr
So we know this stderr is for this command